pkg/plan/recipe/install_plans.go

```diff
@@ -229,8 +230,13 @@ func BuildK8SPlan(kubernetesVersion string, kubeletNodeIP string, seLinuxInstall
 	}
 	processAdditionalArgs := func(cmdline string) string {
 		result := cmdline
+		strs := []string{}
 		for name, value := range extraArgs {
-			result = fmt.Sprintf("%s --%s=%s", result, name, value)
```

Comment:
[disclaimer: this didn't change in this PR & i haven't looked deeper into this]

is it possible for `value` to contain whitespace?